### PR TITLE
utils:kata-manager: Don't check containerd when `only_kata` is enabled

### PR DESCRIPTION
**Pull request by darfux**
Fixes: #6071

Signed-off-by: Li Yuxuan <liyuxuan.darfux@bytedance.com>

### CODE REVIEW

File: utils/kata-manager.sh [hunk 1]
```diff
@@ -269,16 +269,20 @@ containerd_installed()
 
 pre_checks()
 {
+	local only_kata="${1:-}"
```
Comment (amshinde) [inline utils/kata-manager.sh:272]: Maybe  use `skip-containerd` here instead of `only_kata` to make things clearer.

File: utils/kata-manager.sh [hunk 2]
```diff
@@ -269,16 +269,20 @@ containerd_installed()
...
 
 	command -v "${kata_shim_v2}" &>/dev/null \
 		&& die "Please remove existing $kata_project installation"
 
-	local ret
+	if [ "$only_kata" != "true" ]; then
+		local ret
 
-	{ containerd_installed; ret=$?; } || true
+		{ containerd_installed; ret=$?; } || true
```
Comment (amshinde) [inline utils/kata-manager.sh:282]: Check the use of space here instead of tab.

Comment (darfux) [reply utils/kata-manager.sh:282]: Here is tab actually 🧐 I'm not sure why it was highlighted

**Comment by darfux** [comment]
Hi @amshinde , I've resolved the conflicts. PTAL😄

**Pull request closed without merging**